### PR TITLE
Sa/updating to mvi detail screen

### PR DESCRIPTION
**Pull request by SultanArshad**
Updating detail screen to MVI design pattern

### CODE REVIEW

**Review by xxfast — CHANGES_REQUESTED** [review]
Looks like this commit includes changes from #16. As previously mentioned, we want each PR to be concise to it's purpose outlined in the title and description

File: app-kotlin/src/main/kotlin/au/com/gridstone/trainingkotlin/screens/details/DetailViewPresenter.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  private val spDefenseView: TextView = view.findViewById(R.id.sp_defense_value)
+  private val speedView: TextView = view.findViewById(R.id.speed_value)
+  private val hpView: TextView = view.findViewById(R.id.hp_value)
+  private val imageView: ImageView = view.findViewById(R.id.image)
+
+  init {
+    toolbar.title = pokemon.name
+    toolbar.setNavigationOnClickListener {
+      router.popCurrentController()
+    }
```
[the remark this answers is not in the document]
Comment (xxfast) [reply app-kotlin/src/main/kotlin/au/com/gridstone/trainingkotlin/screens/details/DetailViewPresenter.kt:33]: is this addressed?

**Review by xxfast — CHANGES_REQUESTED** [review]
Almost there 👍

**Comment by SultanArshad** [comment]
> Almost there 👍

Please check conversation resolved

**Review by xxfast — APPROVED** [review]
Nice work 👍

**Pull request merged**